src/com/dotmarketing/business/web/WebContext.java

```diff
@@ -0,0 +1,48 @@
+package com.dotmarketing.business.web;
+
+import com.dotcms.AppContext;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+/**
+ * Created by freddyrodriguez on 31/8/16.
+ */
+public class WebContext implements AppContext {
+
+    private HttpSession session;
+
+    private WebContext(HttpServletRequest request){
+        this.session = request.getSession();
+    }
+
+    public static WebContext getInstance(HttpServletRequest request){
+        return new WebContext(request);
+    }
+
+    @Override
+    public <T> T getAttribute(String attributeName) {
```

Comment:
you can do this:
String s = context.getAttribute("AAA");

instead
String s = (String) context.getAttribute("AAA");